### PR TITLE
Add integration instructions for native ads

### PR DESCRIPTION
**Pull request by anton-kapralov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,6 +181,26 @@ class MainActivity : AppCompatActivity(), SdkInitializationListener {
 
     MoPub.initializeSdk(this, sdkConfig.build(), this)
   }
+  
+  // Native Ads
+  fun setUpNativeAds() {
+    moPubNative = MoPubNative(
```
Comment (zdyn) [inline README.md:187]:
```suggestion
    val moPubNative = MoPubNative(
```

File: README.md [hunk 2]
```diff
@@ -179,6 +181,26 @@ class MainActivity : AppCompatActivity(), SdkInitializationListener {
...
+      /* context=*/ this, MOPUB_AD_UNIT_ID, nativeNetworkListener
```
Comment (zdyn) [inline README.md:188]: Very minor/subjective:
```suggestion
      /* context= */ this, MOPUB_AD_UNIT_ID, nativeNetworkListener
```

File: README.md [hunk 3]
```diff
@@ -315,6 +385,7 @@ successful integration.
 | `liftoff-interstitial-video-test` | Interstitial   | VAST video                 |
 | `liftoff-interstitial-html-test`  | Interstitial   | HTML video                 |
 | `liftoff-rewarded-video-test`     | Rewarded Interstitial | VAST video
+| `liftoff-native-test`             | Native         | Native
```
Comment (zdyn) [inline README.md:388]: Can you reformat this table so that the columns line up, and add the final pipe to the last 2 lines?

File: README.md [hunk 4]
```diff
@@ -357,6 +428,14 @@ Custom event data: `{"adUnitID": "LIFTOFF_REWARDED_INTERSTITIAL_AD_UNIT_ID"}`
 
 ![](https://user-images.githubusercontent.com/573865/116912863-9fbd7000-abfd-11eb-826f-faa17392aeaa.png)
 
+#### Native
```
Comment (zdyn) [inline README.md:431]: Update the list above to include native:
```
The screenshots below show example configurations for Liftoff banner/medium
rectangle, standard interstitial, rewarded interstitial, and native line items.
```

File: README.md [hunk 5]
```diff
@@ -179,6 +181,26 @@ class MainActivity : AppCompatActivity(), SdkInitializationListener {
...
+          .textId(R.id.nativeAdDescription)
+          .callToActionId(R.id.nativeAdCTA)
+          .iconImageId(R.id.nativeAdIcon)
+          .mainImageId(R.id.nativeAdMainImage)
+          .build()
+      )
+    )
+    moPubNative.makeRequest()
+  }
+
```
Comment (zdyn) [inline README.md:203]: Extraneous newline.

**Review by zdyn — CHANGES_REQUESTED** [review]

**Comment by anton-kapralov** [comment]
Thanks!

**Review by zdyn — APPROVED** [review]
LGTM!

**Pull request merged**